data/core/command.lua

```diff
@@ -45,7 +45,6 @@ end
 function command.add(predicate, map)
   predicate = command.generate_predicate(predicate)
   for name, fn in pairs(map) do
-    assert(not command.map[name], "command already exists: " .. name)
```

Comment:
What do you think of replacing this with a `core.log_quiet` so when looking at the log one can at least know if a command was overwritten?

Reply:
Yeah, makes sense.